config/gulp/javascript.js

```diff
@@ -27,6 +27,42 @@ gulp.task('eslint', function (done) {
 
 });
 
+gulp.task('copy-docs-javascript', function (done) {
+
+  dutil.logMessage(task, 'Copying JS from js/');
+
+  var stream = gulp.src('./js/**/*')
+    .pipe(gulp.dest('assets/js/'));
+
+  return stream;
+
+});
+
+gulp.task('copy-jquery-javascript', function (done) {
```

Comment:
What do you mean, as part of `package.json`? (Though, now that you point it out, I realised that we don't actually need to be copying any JS (other than `uswds.js`) anywhere - the build process is going to pull the vendor JS from `node_modules` during the compilation stage, so it doesn't need to be in `assets/js`. Silly me. I'm going to fix that.)